go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/GoogleCloudPlatform/compute-image-tools
```

Comment:
so for local development I used go mod to handle all dependencies since I didn't see an existing vendor tree or a dep object (otherwise there's no way to build/test locally, the doc suggested go get however it throws an error)

```
senlu@senlu:~$ go get github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisy
# github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:88:75: undefined: "google.golang.org/api/compute/v0.beta".GuestAttributes
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:107:11: c.OrderBy undefined (type *"google.golang.org/api/compute/v1".FirewallsListCall has no field or method OrderBy)
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:109:11: c.OrderBy undefined (type *"google.golang.org/api/compute/v1".ImagesListCall has no field or method OrderBy)
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:111:11: c.OrderBy undefined (type *"google.golang.org/api/compute/v1".MachineTypesListCall has no field or method OrderBy)
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:113:11: c.OrderBy undefined (type *"google.golang.org/api/compute/v1".ZonesListCall has no field or method OrderBy)
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:115:11: c.OrderBy undefined (type *"google.golang.org/api/compute/v1".InstancesListCall has no field or method OrderBy)
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:117:11: c.OrderBy undefined (type *"google.golang.org/api/compute/v1".DisksListCall has no field or method OrderBy)
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:1043:91: undefined: "google.golang.org/api/compute/v0.beta".GuestAttributes
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/test_client.go:104:82: undefined: "google.golang.org/api/compute/v0.beta".GuestAttributes
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/test_client.go:480:95: undefined: "google.golang.org/api/compute/v0.beta".GuestAttributes
work/src/github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute/compute.go:117:11: too many errors
```

let me know if you already have a solution for this or perfer not commit the gomod files.

Reply:
Looks like you had old deps in your tree that's why there were errors? I'm not against setting up gomod, just haven't got around to doing it for this repo yet.